pkgs/top-level/all-packages.nix

```
@@ -273,7 +273,7 @@ in
   fetchhg = callPackage ../build-support/fetchhg { };
 
   # `fetchurl' downloads a file from the network.
-  fetchurl = import ../build-support/fetchurl {
+  fetchurl = callPackage ../build-support/fetchurl {
```

We don't need the full `callPackage` infrastructure (the automatic finding of args), so we can just make this overridable instead.

```suggestion
  fetchurl = makeOverridable (import ../build-support/fetchurl) {
```

---

Awsome, I've applied that, thanks!